specification/cognitiveservices/data-plane/FormRecognizer/readme.md

```diff
@@ -15,6 +15,13 @@ openapi-type: data-plane
 
 ``` yaml
 
+tag: release_2_0_preview
```

Comment:
does this need to be release_2_0_GA?

Reply:
It seems this entire new block is not necessary (line 18 to 24)

Reply:
Take a look at the collapsed tag above, that one targets GA and is just `release_2_0`

Reply:
Following Text Analytics format where preview releases have `preview` tagged on it and GA releases do not: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/cognitiveservices/data-plane/TextAnalytics#releases

Reply:
Sorry, didn't see your comment @lmazuel - do we not want to allow generation of the 2.0-preview?